yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+---
```

Comment:
I expect to see a number of updates for this type of schema. Maybe it's worth thinking about schema versioning and schema upgrades like what we added to APDB recently (https://dmtn-269.lsst.io/).

Reply:
There will be a million updates to this, yes.  I am hoping that @JeremyMcCormick will come up with an Alembic-based way to deal with versioning and upgrades.